src/utils/material-about.js

```diff
@@ -1,16 +1,20 @@
 import { config } from '../configs';
 
-const { languages } = config;
+const { languages, IMG_URL } = config;
 
-export const getInitialValuesForMaterialsAdd = () => ({
-  img: '',
-  uaTitle: '',
-  enTitle: '',
-  uaText: '',
-  enText: ''
+export const getInitialValuesForMaterialsBlock = ({
```

Comment:
I pretty sure this utility function will never been used anywhere except materialsForm component, so you can reduce the name of it.